controls/imageeditor/loading-image.md

```diff
@@ -1,53 +1,53 @@
 ---
 title: Display Images
 page_title: .NET MAUI ImageEditor Documentation - Display Images
-description: "Review the options to display images in the .NET MAUI ImageEditor control."
+description: Learn what are the options to display images in the Telerik UI for .NET MAUI ImageEditor control.
 position: 2
 slug: imageeditor-loading-image
 ---
 
-# Display Images in .NET MAUI ImageEdiotor
+# Display Images in .NET MAUI ImageEditor
 
 ImageEditor control enables you to visualize images using the following property:
 
 * `Source`(of type `Microsoft.Maui.Controls.ImageSource`): Specifies the source of the image. For more details about the Source property check the [Images in .NET MAUI](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/image?view=net-maui-7.0) article.
 
-The images could be loaded from:
+The images can be loaded from:
 
 * `File`
 * `Uri`
 * `Stream`
 * `Resources`
 
-**Load images for File**
+**Load Images for File**
 
 <snippet id='imageeditor-load-image-from-file'/>
 
-**Load images for Uri**
+**Load Images for Uri**
 
 ```XAML
 <telerik:RadImageEditor x:Name="imageEditor" 
                         Source="https://raw.githubusercontent.com/telerik/maui-samples/main/Samples/ControlsSamples/Resources/Images/borderconfigurationavatar.png" />
         
 ```
 
-**Load images for Stream**
+**Load Images for Stream**
 
 <snippet id='load-image-from-stream'/>
 
-**Load images for Resources**
+**Load Images for Resources**
 
 <snippet id='imageeditor-toolbar-styling'/>
 
 ## Loading Template
 
-By default a busy indicator is shown when loading an image. You can change the indicator template using the following property:
+By default a busy indicator is illustrated when loading an image. You can change the indicator template using the following property:
```

Comment:
You are getting a warning because of the passive voice, not because of the word "show". Don't use "illustrated" in this context.